app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,18 @@
+class VotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    review = Review.find(params[:review_id])
+    vote = review.votes.find_or_initialize_by(user: current_user)
+    vote.assign_attributes(vote_params)
+    vote.save
```

Comment:
We were thinking it could actually be annoying to get a flash message if you've already voted on something. Is it okay to leave it without?